index.html

```diff
@@ -1086,6 +1085,32 @@ <h2>Help your friends find some good learning resources</h2>
 
 
             <!--contributor card end-->
+             <!-- ________Aarushi Contributor card START ________  -->
+             <div class="card">
+                <p class="name">Aarushi</p>
+                <p class="contact">
+                    <i class="fab fa-linkedin-in"></i>
+                    <a href="https://www.linkedin.com/in/aarushi-sharma-1b24211a9/" target="_blank">Aarushi Sharma</a>
+                </p>
+                <p class="about">I am a Computer Science(AI&ML) Student from Lucknow,India. Presently, I am familiar with Java, Python 
+                    and MySQL, with a little bit of front-end here and there. Currently, I am learning ML and am open to contributing in open source projects.
+                    Besides coding, I like to read and play basketball.</p>
+                <div class="resources">
+                    <p>3 Useful Java Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://dev.java/learn/getting-started-with-java/" target="_blank" title="First Resource">Java Documentation</a>
+                        </li>
+                        <li>
+                            <a href="https://www.geeksforgeeks.org/java/" target="_blank" title="Second Resource">Step by step tutorials</a>
+                        </li>
+                        <li>
+                            <a href="https://www.youtube.com/watch?v=rZ41y93P2Qo&list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ" target="_blank" title="Third resource">DSA Java course by one of the best EdTech YouTuber.</a>
```

Comment:
give a descriptive title to the resource, like  youtube channel name

Reply:
Sure!!